capa/ida/plugin/view.py

```diff
@@ -178,6 +178,9 @@ def build_context_menu(o, actions):
 
 
 class CapaExplorerRulgenPreview(QtWidgets.QTextEdit):
```

Comment:
missing an e in "Rule", but no biggie and not related here at all

Reply:
```suggestion
class CapaExplorerRulegenPreview(QtWidgets.QTextEdit):
```

references will need fixup too, though

Reply:
let's save this for #488.